File: Competitive Programming 3/Section 1.3.3/12372 - Packing for Holiday/C++/main.cpp
#include <iostream>

using namespace std;

int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);
    int t,x;
    cin >> t ; 
    for(int cs = 1 ; cs<=t ; cs++){
        string ans = "good";
        for(int i = 0 ; i < 3 ; i ++){
            cin >> x ; 
            if(x>20){
                ans="bad";
            }
        }
        cout << "Case " << cs << ": " << ans << "\n";
    }
    return 0;
}

File: Competitive Programming 3/Section 1.3.3/12372 - Packing for Holiday/Java/Main.java
import java.util.Scanner;

public class Main {
    static Scanner s = new Scanner(System.in);
    public static void main(String[]args){
        int t = s.nextInt();
        for(int cs = 1 ; cs <= t ; cs++){
            String ans = "good";
            for(int i = 0 ; i < 3 ; i ++){
                if(s.nextInt()>20)
                    ans="bad";
            }
            System.out.printf("Case %d: %s\n",cs,ans);
        }
    }
}